src/gui/command.ts

```diff
@@ -127,19 +130,22 @@ export class FindElementsFileCommand {
             },
             async (progress) => {
                 try {
-                    let execResult = await execShell(
-                        `${result || "python"} ${join(__dirname, "assets", "find_element_file.py")}`
-                    );
+                    let execResult = await execShell(`${pythonPath} ${join(__dirname, "assets", "find_element_file.py")}`);
                     if (execResult.startsWith("Path: ")) {
                         updateFilePath(execResult.substring(6));
-                        window.showInformationMessage(l10n.t("Visual element descriptors file was found and updated in workspace settings"));
+                        window.showInformationMessage(
+                            l10n.t("Visual element descriptors file was found and updated in workspace settings")
+                        );
                     } else if (execResult) {
                         window.showErrorMessage(execResult);
                     } else {
-                        window.showErrorMessage(l10n.t("Can't find visual element descriptors file with the provided environment"));
+                        window.showErrorMessage(
+                            l10n.t("Can't find visual element descriptors file with the selected environment")
+                        );
                     }
                 } catch (error) {
-                    window.showErrorMessage(l10n.t("Can't find visual element descriptors file with the provided environment"));
+                    console.info(error);
```

Comment:
console.info ?
We'll have to start using LogOutputChannel

Reply:
Will copy what you have done on config